src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsChannelMappingTest.java

```diff
@@ -234,4 +236,16 @@ void testIsPaperCaseAttending() {
 
         assertThat(result).isFalse();
     }
+
```

Comment:
Add a test for feature flag enabled/disabled with expected code blocks being used in each case

Reply:
added test cases to reflect this